### DISCUSSION TITLE
Documentation for deploying to different PaaS solutions

### DISCUSSION THREAD

**Discussion by aleccool213** [mealie-recipes/mealie#693] [Feature Request]
Want to try this out but dont want to go through the pain of deploying to Heroku myself.

**Comment by hay-kot** [comment]
This likely won't be addressed until authentication is introduced. Ideally, I'd love to have a single click install with Heroku or Digital Ocean.

**Comment by michael-genson** [comment]
As of now Mealie's only supported deployment method is via Docker. We will accept PRs for additional deployment documentation.

I'm currently going through some old discussions, so apologies for the necro.
